keyboards/satt/vision/vision.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_default( \
```

Comment:
As there only seems to be one layout possible, this should be named `LAYOUT`.

```suggestion
#define LAYOUT( \
```